quantum-adapters/kiwoom-adapter/src/kiwoom_api/api/stock.py

```
@@ -311,33 +331,567 @@ async def api_fn_ka90003(
+    **응답 데이터:**
+    - **raw_data**: 키움 API 원본 응답 (전체 필드 포함)
+    - **market_info**: 구조화된 시세표성정보
+      - **basic_info**: OHLCV 기본 시세
+      - **orderbook**: 매도/매수 호가 배열
+      - **lp_info**: LP 매도/매수 호가
+      - **market_data**: 시총, 외국인비율, 공매도비율, 체결강도
+
+    **실시간 시세표성정보를 제공하는 핵심 API입니다**
+    """
+    try:
+        request_time = datetime.now().strftime("%Y%m%d%H%M%S")
```

Ruff (0.12.2)

Undefined name `datetime`

(F821)